etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml

```diff
@@ -0,0 +1,21 @@
+# NOTE: To learn more about the fields, hover over their names.
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  bmj_2022:
+    variables:
+      essential_medicine_list:
+        title: Statin inclusion on essential medicine list
+        unit: arbitrary units
+        short_unit: au
+        description_short: Denotes whether statins were included in national Essential Medicines Lists (EMLs).
+        display:
+          numDecimalPlaces: 0
+
+      statin_use:
+        title: Statin use
+        unit: "per 1000 population ≥ 40 years old"
+        short_unit: ""
+        description_short: Based on purchases made by stores and hospitals in 52 countries and, in special cases like the UK, using data about the volume given to patients.
```

Comment:
Maybe you can include a text in `description_processing` mentioning that data for this variable in 2019 also includes 2020

Reply:
good point! thanks!! :)